src/routes/plans/[id]/+page.svelte

```diff
@@ -137,6 +141,16 @@
       .getResourceTypes($plan.model_id, data.user)
       .then(initialResourceTypes => ($resourceTypes = initialResourceTypes));
   }
+  $: if (data.initialPlanSnapshotId !== null) {
```

Comment:
Do we care if the snapshotId doesn't map to a snapshot? Like should we warn the user if the snapshot does not exist? Otherwise you might think you're viewing the snapshot. Though I guess the design call for that horizontal message conveying that you're viewing a snapshot so maybe that's enough?

Reply:
hmm yeah! that's a good idea. I still have more work to do for snapshots within the plan page so I could add that message in my next PR?

Reply:
Sounds good to me